server.js

```diff
@@ -16,7 +17,7 @@ require('./models/User')
 app.use(bodyParser.urlencoded({extended:false}));
 app.use(bodyParser.json())
 app.use(require('./routes'));
-app.use(cors());
```

Comment:
Anton when the line "app.use(cors());" was on line 19, I was getting a CORS error in the console. Moving it to line 10 fixes the CORS error.

Reply:
Perfect!